DigitalExcellence/dex-frontend
# Share icons

Created share button on project details modal copying the user's current URL to the user's clipboard.

## Description

Giving the user the possibility to share the currently opened project. By clicking on the share button the user's URL gets copied to her/his clipboard.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Go to the projects overview.
2. Select a project so the projects detail modal will open.
3. Click on the 'Copy url' button.
4. Try pasting your clipboard in a fresh tab.
5. Run the pasted clipboards text and see if it goes to the selected project from step 2.

## Link to issue

Closes: #442
